src/components/Clickable.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import clsx from 'clsx'
+
+export function Clickable(
```

Comment:
not sold on the name as it doesn't reflect the keyDown aspect, but I don't have a bettername. ClickOrEnter?

Reply:
This was a copy and paste from the renderer. I agree! `<PretendButton />` 🤔